test/extended/networking/util.go

```diff
@@ -184,6 +188,38 @@ func launchWebserverPod(f *e2e.Framework, podName string, nodeName string) (ip s
 	return
 }
 
+func launchWebserverService(f *e2e.Framework, serviceName string, nodeName string) (serviceIP string) {
```

Comment:
Consider renaming the return value to serviceSocket, since 'socket' is the combination of hostname/ip:port according to rfc793.